libutils/platform.h

```diff
@@ -178,6 +178,9 @@ struct utsname
 #endif
 
 #include <signal.h>
+#ifdef __ANDROID__
+#include <android/log.h>
+#endif
```

Comment:
Should this go right next to the `ifdef` below? I take it it defines the same? Or maybe it should go to `logging.h`?

Reply:
Checking why I needed this at all. Then will move and comment appropriately.

Reply:
Not sure why I had this. It doesn't seem to be needed so I removed it.